testcases/single_cell_ispol/README

```diff
@@ -0,0 +1,22 @@
+# Set DOMAINS DIRECTORY for ISPOL (67.9 S, -54.4 E)
```

Comment:
This is specific to anvil so please remove the README

Reply:
Removed README